V5_README.md

```diff
@@ -146,4 +147,11 @@ Uploading from the command line
 
 While there is no dedicated command to upload documents, this library does
 include the command `lumi-api`, which can be used to access the API in general
-and to upload documents in particular.  Run `lumi-api -h` for more information.
\ No newline at end of file
+and to upload documents in particular.  Run `lumi-api -h` for more information.
+
+Examples:
+```
+lumi-api -b https://analytics.luminoso.com/api/v5/ -t my_token get /projects
+lumi-api -b https://analytics.luminoso.com/api/v5/ -t my_token get /projects/my_project_id/concepts/match_counts
+lumi-api -b https://analytics.luminoso.com/api/v5/ -t my_token post /projects/my_project_id/concepts/saved -j '{"concepts": [{"texts": ["My new concept text"]}]}'
+```
```

Comment:
To be honest, we need a *lot* more documentation than this.  Rob has been having trouble figuring out the right sequence for creating a project, uploading documents to it, and building it, and I think those are at a bare minimum also things that need to be covered.

Or, to look at it another way: the examples of using the client interactively show how to get a project list, how to create a project, upload to that project, build that project, wait for that project to build, and get concepts from that project.  The documentation for the CLI client should be at least that complete.  To say nothing of saving a token, printing the output in CSV format...